ledger/tracker.go

```diff
@@ -467,11 +467,15 @@ func (tr *trackerRegistry) commitRound(dcc *deferredCommitContext) {
 	tr.mu.Lock()
 	tr.dbRound = newBase
 	for _, lt := range tr.trackers {
-		lt.postCommit(tr.ctx, dcc)
+		lt.postCommit(tr.ctx, dcc, true)
 	}
 	tr.lastFlushTime = dcc.flushTime
 	tr.mu.Unlock()
 
+	for _, lt := range tr.trackers {
+		lt.postCommit(tr.ctx, dcc, false)
```

Comment:
maybe split `postCommit` into two methods like `postCommitSync`, `postCommitAsync` ?

Reply:
agreed, it is kind of confusing to have two modes for postCommit versus a second method, for example would make it more clear by seeing that accountUpdates has an empty implementation. a name like postCommitAsync doesn't sound exactly right though, trackerRegistry.commitRound() is still going to wait for the catchpoint to be generated.. maybe postCommitUnlocked?

Reply:
agree, `postCommitUnlocked` sounds better